x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/integration/BasicDistributedJobsIT.java

```diff
@@ -68,6 +70,11 @@ public void testFailOverBasics() throws Exception {
         client().execute(OpenJobAction.INSTANCE, openJobRequest).actionGet();
         awaitJobOpenedAndAssigned(job.getId(), null);
 
+        // always default delayed allocation to 0 to make sure we have tests are not delayed
+        client().admin().indices().updateSettings(new UpdateSettingsRequest(".ml-*")
+            .settings(Settings.builder().put(UnassignedInfo.INDEX_DELAYED_NODE_LEFT_TIMEOUT_SETTING.getKey(), 0).build()))
+            .actionGet();
```

Comment:
Since both classes where this has been added extend `BaseMlIntegTestCase` it's probably worth putting this code in a utility method in that base class.

I also wonder if `NetworkDisruptionIT` might need it, depending on which random form of network disruption is chosen.  `NetworkDisruptionIT.testJobRelocation()` contains `ensureGreen()` calls.  It could be that for certain random number sequences these require a shard to reassign to attain green status.